special/OreDictEntryManager.php

```diff
@@ -123,13 +123,15 @@ private function createEntry(FormOptions $opts){
 	private function updateEntry(FormOptions $opts){
 		$dbw = wfGetDB(DB_MASTER);
 		$stuff = $dbw->select('ext_oredict_items', '*', array('entry_id' => $opts->getValue('entry_id')));
-		$result = $dbw->update('ext_oredict_items', array(
+		$ary = array(
 			//'tag_name' => $opts->getValue('tag_name'),
 			'item_name' => $opts->getValue('item_name'),
 			'mod_name' => $opts->getValue('mod_name'),
 			'grid_params' => $opts->getValue('grid_params'),
 			'flags' => $opts->getValue('flags')
-		), array('entry_id' => $opts->getValue('entry_id')));
+		);
+		$tableName = $dbw->tableName('ext_oredict_items');
```

Comment:
I'm not a database developer... What does that mean exactly? :stuck_out_tongue: 

It's used all over this extension, at least in this class. If we shouldn't do that, what should we do instead? Just call it as `$dbw->delete('ext_oredict_items', array('entry_id' => $opts->getValue('entry_id')));`?